pathoplexus_app/README.md

```diff
@@ -0,0 +1,15 @@
+# Pathoplexus_app
+
+This directory is a tiny Helm chart used to deploy instances of Pathoplexus. It creates a kubernetes resource for an ArgoCD `Application`. _That application_ deploys the (large) _Loculus_ Helm chart. The version (SHA) of Loculus to be used is defined by the `Values.yaml` in this directory.
+
+All the specific Loculus configuration is defined in `loculus_values` in the parent directory. The end result is that each version (commit) of this _pathoplexus_ repository contains a reference, in this `values.yaml` to a particular version of Loculus, which is used:
+  - to combine with the `monorepo` directory to build the Pathoplexus website image
+  - to specify the version of the Loculus Helm chart to deploy
+  - to specify what versions of the Loculus component to deploy
```

Comment:
What's the difference between them?

Reply:
This was a typo and I should have written "components", but actually a better word is "images" which I'll change this to, thanks